misc/docker-integration/integration_test.go

```diff
@@ -149,6 +149,7 @@ func startGnoland(t *testing.T) {
 		"docker", "run",
 		"-d",
 		"--name", gnolandContainerName,
+		"-e", "GNOROOT=/opt/gno/src",
```

Comment:
it's a side effect of changing the relative base directory of stdlibs to `GnoRootDir` in app.go:
```diff
modified   gno.land/pkg/gnoland/app.go
@@ -33,21 +67,21 @@ func NewApp(rootDir string, skipFailingGenesisTxs bool, logger log.Logger, maxCy
 	baseKey := store.NewStoreKey("base")
  
 	// Construct keepers.
 	acctKpr := auth.NewAccountKeeper(mainKey, ProtoGnoAccount)
 	bankKpr := bank.NewBankKeeper(acctKpr)
-	stdlibsDir := filepath.Join("..", "gnovm", "stdlibs")
+	stdlibsDir := filepath.Join(cfg.GnoRootDir, "gnovm", "stdlibs")
    vmKpr := vm.NewVMKeeper(baseKey, mainKey, acctKpr, bankKpr, stdlibsDir, cfg.MaxCycles)
 ```
Normally this directory is guessed using the go binary, but on docker integration go binary doesn't exist, so he is unable to guess the rootdir.
However you make me think that this should cause some issues on actual docker instance if updated, I should probably add this to the `Dockerfile` as ENV variable instead of setting it here.
One (simpler) other solution should be to fallback on ".." like before, but I think we should get ride of this relative path once and for all.

Reply:
dockerfile change looks good. :+1: